src/UI/Implementation/Component/Panel/Report.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Panel;
+
+use ILIAS\UI\Component as C;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
+
+/**
+ * Class Panel
+ * @package ILIAS\UI\Implementation\Component\Panel
+ */
+class Report extends Panel implements C\Panel\Report {
+	use ComponentHelper;
+
+	/**
+	 * @var \ILIAS\UI\Component\Panel\Sub[]
+	 */
+	private  $sub_panels;
+
+	/**
+	 * @param string $title
+	 * @param \ILIAS\UI\Component\Panel\Sub[]
+	 */
+	public function __construct($title,$sub_panels) {
```

Comment:
The `Report` class extends `Panel`, but doesn`t invoke its constructor and thus leaves the inherited property `content` uninitialized, which makes `Panel::getContent` promise something it can't fullfill. Overall it looks to me as if `Report` is basically `Panel` with an additional constraints on the content. Maybe we should model it just like that.